client/src/components/TransferForm.vue

```diff
@@ -17,18 +19,32 @@
       </md-select>
     </md-field>
 
-    <div v-if="transferParams[transferModel] !== undefined">
+    <div v-if="Object.keys(allTransferData[transferChoice]).length !== 0 &&
+      transferChoice !=='CAMB'">
       <InputField
-        v-for="(input, inputName) in transferParams[transferModel]"
+        v-for="(input, inputName) in allTransferData[transferChoice]"
         :labelHtml="'<label>' + inputName + '</label>'"
         :key="inputName"
         :step="1"
-        :currentValue="transferParams[transferModel][inputName]"
-        v-on:updateCurrentValue="createSetCurrentValueFunc(transferModel, inputName)($event)"
+        :currentValue="allTransferData[transferChoice][inputName]"
+        v-on:updateCurrentValue="createSetCurrentValueFunc(transferChoice, inputName)($event)"
       />
     </div>
 
-    <md-checkbox v-model="takahashiChoice">
+    <DoubleField
+      :param="'lnk_min'"
+      v-model="transferData.lnk_min"
+    />
+    <DoubleField
+      :param="'lnk_max'"
+      v-model="transferData.lnk_max"
+    />
+    <DoubleField
+      :param="'dlnk'"
+      v-model="transferData.dlnk"
+    />
```

Comment:
These all have mins/maxes defined in forms.py from original halomod app. Would be good to have that validation.

Reply:
Validation was added.